index.html

```diff
@@ -0,0 +1,167 @@
+      </div>
+
+      <div class="programs">
+        <div class="consult">
+          <i class="fas fa-book-reader "></i>
+
+          <h3 class="bussiness-h1">Lecture</h3>
+          <p class="bussiness-para">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Fugit libero sit in necessitatibus, ad
+            sequi?
+          </p>
+        </div>
+        <div class="design">
+          <i class="fab fa-accusoft"></i>
+          <h3 class="bussiness-h2">Exihibition</h3>
+          <p class="bussiness-para">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Fugit libero sit in necessitatibus, ad
+            sequi?
+          </p>
+        </div>
+        <div class="testing">
+          <i class="far fa-comments"></i>
+          <h3 class="bussiness-h3">Forum</h3>
+          <p class="bussiness-para">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Fugit libero sit in necessitatibus, ad
+            sequi?
+          </p>
+        </div>
+        <div class="commssioning">
+          <i class="fas fa-code-branch light-grey"></i>
+          <h3 class="bussiness-h4">Workshop</h3>
+          <p class="bussiness-para">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Fugit libero sit in necessitatibus, ad
+            sequi?
+          </p>
+        </div>
+        <div class="training">
+          <i class="fas fa-chart-line light-grey"></i>
+          <h3 class="bussiness-h5">Ignite</h3>
+          <p class="bussiness-para">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Fugit libero sit in necessitatibus, ad
+            sequi?
+          </p>
+        </div>
+
+      </div>
+      <div class="detail-prog">
+        <a class="join-mobile" href="">SEE THE WHOLE PROGRAM</a>
+        <a class="join-desk" href="">SEE THE WHOLE PROGRMA</a>
+      </div>
+
+    </section>
+    <section class="feature-speaker">
+    </section>
+    <section class="partners-d">
+      <div class="h2-hr-part">
+        <h2 class="part-h2">Partners</h2>
+        <hr class="partner-line">
+      </div>
+      <ul class="partners-lst">
+        <li><img src="/assets/fb-logo.png" alt="Facebook"></li>
+        <li><img src="/assets/google-logo.jfif" alt="google"></li>
+        <li><img src="/assets/mozila-logo.png" alt="mozila-logo"></li>
+        <li><img src="/assets/twitter-logo.png" alt="twitter"></li>
+      </ul>
+    </section>
+    <div class="container-reserved">
+      <div class="some-right">
+        <img class="some-right-image" src="assets/creative_commons.jpeg" alt="">
+        <div class="right-reserved">
+          <h6>2023 creative common Korean, Some rights are resreved</h6>
+          <p class="right-reserved-p">Lorem ipsum dolor sit amet consectetur adipisicing elit. Saepe corporis pariatur minima placeat molestias,
+            numquam quam vel! Repudiandae, quisquam quidem.
+          </p>
+        </div>
+      </div>
+    </div>
```

Comment:
- [ ] Please make this `div` elements a `footer` because it will make your HTML code semantic